allegro/src/bitmap.rs

```diff
@@ -36,6 +51,19 @@ impl Bitmap
 		}
 	}
 
+	/// This method is used to load a bitmap file into a Bitmap object.
+	/// Takes a string path_to_map that points to a bitmap file.
+	/// # Examples
+	///  ```
```

Comment:
Extra leading space here.

Reply:
I am not able to see it the extra leading space myself..

Reply:
There's two spaces between /// and ```, one is extraneous.